plugins/module_utils/redfish_utils.py

```diff
@@ -2061,18 +2078,9 @@ def get_nic_inventory(self, resource_uri):
             nic_list.append(nic[u'@odata.id'])
 
         for n in nic_list:
-            nic = {}
-            uri = self.root_uri + n
-            response = self.get_request(uri)
-            if response['ret'] is False:
-                return response
-            data = response['data']
-
-            for property in properties:
```

Comment:
Since this was moved to the new `get_nic` method, it looks like we can delete the `properties` list defined earlier in this method.